src/main/java/helper/ElementHelper.java

```diff
@@ -16,4 +17,12 @@ public static boolean isElementDisplayed(WebElement webElement) {
             return false;
         }
     }
+
+    public static boolean isTicketDisplayed(By selector) {
```

Comment:
dont write a method like this in ElementHelper, ElemenetHelper is not something relating to ticket

Reply:
Fixed. Please check again